src/server/tests/sample.test.js

```diff
@@ -0,0 +1,5 @@
+describe('Sample Test', () => {
```

Comment:
Since you added a route test, do we need this sample test? If not, lets remove it.

Reply:
I would like to leave it for now because I know it works, at least until I can figure out how to get Jest running inside of Docker